crawler/crawlutils.py

```diff
@@ -287,6 +288,42 @@ def snapshot_generic(
     ) as emitter:
         snapshot_single_frame(emitter, features,
                               options, crawler, inputfile)
+def snapshot_mesos(
+    crawlmode=Modes.MESOS,
+    inurl=['stdin://'],
```

Comment:
Can you make this inurl an option? This argument only applies to mesos but its therefor every single feature collected. Ideally,  it should only apply  to mesos. Take a look at the extra options for the file feature (exclude_dirs for example).